src/test/java/gov/healthit/chpl/aqa/asserts/ViewVersionPageAsserts.java

```diff
@@ -40,5 +40,26 @@ public void iSeeMergeLinkToMergeTheVersion(final String versionId) {
         ViewVersionPage.mergeVersionLink(getDriver(), versionId).isDisplayed();
         assertTrue(true);
     }
+
+    /**
+     * Assert that "At least one other Version must be selected to merge" is displayed in versions to merge section.
+     * @param versionsToMergeMessage is a section in merge versions
+     */
+    @Then("^I see \"([^\"]*)\" in versions to merge section$")
+    public void iSeeMessageInVersionsToMergeSection(final String versionsToMergeMessage) {
+        ViewVersionPage.versionsToMergeMessage(getDriver()).isDisplayed();
+        assertTrue(true);
```

Comment:
This is wrong. If anything, it should be something like
```suggestion
        assertTrue(PageObject.findMessage(theStringVariable)).isDisplayed());
```
I don't know if that'd work, but it'd be closer